doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+
+:::{admonition} **Note 1: Black Box Definition**
+The use of Black Box here varies subtly from the conventional use within machine learning. Typically, we say a model is a black box if the mechanism by which it makes predictions is too complicated to be interpretable to a human. Here we use black-box to mean that the explainer method doesn't need access to the model internals to be applied.
+:::
+
+## Applications:
+
+**Trust:** At a core level, explainability builds trust in the machine learning systems we use. It allows us to justify their use in many contexts where an understanding of the basis of the decision is paramount.
+
+**Testing:** Explainability can be thought of as an extra form of testing for a model. The insights derived should conform to the expected behaviour. Failure to do so may indicate issues with the model or problems with the dataset it's been trained on.
+
+**Functionality:** Insights can be used to augment model functionality. For instance, providing information on top of model predictions such as how to change model inputs to obtain desired outputs.
+
+**Research:** Explainability allows researchers to understand how and why opaque models make decisions. Helping them understand more broadly the effects of the particular model or training schema they're using.
+
+__TODO__:
+- picture of explainability pipeline: training -> prediction -> insight
+
+:::{admonition} **Note 2: Biases**
+
+Practitioners must be wary of using explainability to excuse incorrect models rather than ensuring their correctness. It is possible to have a correctly trained model on a dataset, but the model doesn't reflect reality because the dataset is either wrong or incomplete. Suppose the insights that explainability generates conform to some confirmation bias of the person training the model. In that case, they will be blind to this issue and instead use these methods to confirm erroneous results. A key distinction is that explainability insights are designed to be faithful to the model they are explaining and not the data. You use an explanation to obtain an insight into the data only if the model is trained well.
```

Comment:
"A key distinction is that explainability insights are designed to be faithful to the model they are explaining and not the data." - I think this sentence is too strong, model depends on data hence explainability depends on data too. I see where you're getting at in that explainer output isn't explaining the data by itself (unless, as you argue, it's a very good model - but this can be contentious too), but maybe we can split out these points a bit more as it seems there are at least 3 things going on in this paragraph? 

Also, what do we mean by "incorrect models"?

I would personally favour rewriting as a bullet point of different scenarios (but feel free to comment, also likely needed intuitive examples and concretes biases and also rephrase what "the right thing" is):
- Practitioner biases make explainer output seem like everything works, but actually either the model or the data isn't doing the "right thing"
- Practitioner biases make explainer output seem like something is wrong, but actually both the model and the data are doing the "right thing"
- Practitioner biases make explainer output seem like something is wrong, but both the model and the data could be "bad" or just one of them.
- Note on explanation being a function of the model (and indirectly the data) but not directly of the data? (So e.g. scenarios like good data + bad model = explanation that doesn't conform to your "valid" biases on what to expect from the good data; bad data + good model = similar story).

Reply:
Yes, I think this note is absolutely key and very important to highlight as abuse of explainers in this context seems to be all over the place. Making the point as simple as possible with some scenarios would be best rather than a high-level, more vague description.

Reply:
I've rewritten the Note on the bias into its own section although I feel I'm struggling to convey everything:

The explanations Alibi's methods provide depend on the model, the data, and - for local methods - the instance of interest. Thus Alibi allows us to obtain insight into the model and, therefore, also the data, albeit indirectly. There are several pitfalls of which the practitioner must be wary.

Often bias exists in the data we feed machine learning models even when we exclude sensitive factors. Ostensibly explainability is a solution to this problem as it allows us to understand the model's decisions to check if they're appropriate. However, human bias itself is still an element. Hence, if the model is doing what we expect it to on biased data, we are venerable to using explainability to justify relations in the data that may not be accurate. 

Machine learning engineers may also have expectations about how the model should be working. An explanation that doesn't conform to their expectations may prompt them to decide the model is "incorrect" erroneously. People usually expect classifiers trained on image datasets to use the same structures humans naturally do when identifying the same classes. However, there is no reason to believe such models should behave the same way we do. 

Interpretability of insights can also mislead. Some insights such as anchors give conditions for a classifiers prediction. Ideally, the set of these conditions would be small. However, when obtaining anchors close to decision boundaries, we may get a complex set of conditions to differentiate that instance from near members of a different class. Because this is harder to understand, one might write the model off as incorrect, while in reality, the model performs as desired.

Reply:
I like the write-up, maybe it can be made more concrete with specific examples?

Let's also think how we best incorporate this into the document @arnaudvl.